### PR TITLE
Hubert / implementing app manager

### PR DESCRIPTION
**Pull request by hubert-deriv**

### CODE REVIEW

File: src/features/dashboard/components/ApiTokenForm/api-token.form.tsx [hunk 1]
```diff
@@ -113,11 +120,9 @@ const ApiTokenForm = (props: HTMLAttributes<HTMLFormElement>) => {
           />
         ))}
       </div>
-      <Text data-testid={'second-step-title'}>
+      <Text as={'p'} type={'paragraph-1'} data-testid={'second-step-title'}>
         Name your token and click on Create to generate your token.
       </Text>
-
-      {/* @Hubert Please apply your own input styles onto this, thanks thanks */}
```
Comment (hubert-deriv) [inline src/features/dashboard/components/ApiTokenForm/api-token.form.tsx:120]: note to self @hubert-deriv : Will have to apply my custom input styles

File: src/features/dashboard/components/AppForm/app-form.module.scss [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  width: 100%;
+  .formContent {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    width: 100%;
+
+    select {
+      padding: 0 30px 0 10px;
```
Comment (hubert-deriv) [inline src/features/dashboard/components/AppForm/app-form.module.scss:72]: replace px with rem function

File: src/features/dashboard/components/Tabs/tabs.module.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+.tabs_list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: rem(2);
+}
+
+.tabs_trigger {
+  padding: 0 20px;
+  height: 45px;
```
Comment (hubert-deriv) [inline src/features/dashboard/components/Tabs/tabs.module.scss:22]: replace with rem function

File: src/pages/dashboard.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import { AppManager } from '@site/src/features/dashboard';
+export default function AppRegistrationPage() {
+  return (
+    <Layout title='Dashboard' description='Manager your apps and api tokens'>
```
Comment (hubert-deriv) [inline src/pages/dashboard.tsx:6]: typo, it's `manage your apps`, not `manager your apps`.

File: src/features/dashboard/components/AppForm/index.tsx [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+  });
+  const { tokens } = useApiToken();
+
+  const adminTokens = useMemo(() => {
+    return tokens.filter((item) => item.scopes.includes('admin'));
+  }, [tokens]);
+
+  return (
+    <React.Fragment>
+      <form role={'form'} className={styles.apps_form} onSubmit={handleSubmit(submit)}>
```
Comment (hubert-deriv) [inline src/features/dashboard/components/AppForm/index.tsx:35]: Is the form role necessary on an form element? I thought the form element has this by default.

**Review by sanjam-deriv — APPROVED** [review]

**Comment by hubert-deriv** [comment]
Note: I also reviewed Michio's changes but cannot approve since it's my own PR, hence this note.

**Pull request merged**